### PR TITLE
chore: improved fastlane usage and scripts

### LINKED ISSUE #455: improve fastlane usage and scripts
**Describe the chore**
improve the usage of fastlane and make scripts more readable

**Would you like to work on the issue?**
YES

### PR DESCRIPTION
**Pull request by adityastic**
Fixes #455 

Changes: 
- improved scripts and fastlane usage

### CODE REVIEW

File: scripts/prep-key.sh [hunk 1]
```diff
@@ -1,13 +1,11 @@
 #!/bin/sh
-set -e
 
-export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
-export PUBLISH_BRANCH=${PUBLISH_BRANCH:-master}
-
-if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge-magic-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-    echo "We decrypt key only for pushes to the master or development branch and not PRs. So, skip."
+# Check if it is not a Pull Request Build
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge-magic-android" ]; then
```
Comment (iamareebjamal) [inline scripts/prep-key.sh]: This will decrypt key for all branches

File: scripts/upload-apk.sh [hunk 2]
```diff
@@ -1,48 +1,37 @@
...
 git config --global user.name "Travis CI"
 git config --global user.email "noreply+travis@fossasia.org"
 
-export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
-export PUBLISH_BRANCH=${PUBLISH_BRANCH:-master}
-
-if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge-magic-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-    echo "We upload apk only for changes in development or master, and not PRs. So, let's skip this shall we ? :)"
+# Check if it is not a Pull Request Build
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge-magic-android" ]; then
```
Comment (iamareebjamal) [inline scripts/upload-apk.sh]: Same

**Comment by adityastic** [comment]
@iamareebjamal have a look

File: scripts/upload-apk.sh [hunk 3]
```diff
@@ -12,7 +12,9 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge
     exit 0
 fi
 
-./gradlew bundleRelease
+if [[ "$TRAVIS_BRANCH" != "$PUBLISH_BRANCH" ]]; then
```
Comment (iamareebjamal) [inline scripts/upload-apk.sh]: We always want to build aab

Comment (adityastic) [reply scripts/upload-apk.sh]: great, I've removed the unsigned and unaligned conditions as well as they are not being created

**Pull request merged**